map.go

```diff
@@ -160,6 +184,9 @@ type Map struct {
 	pinnedPath string
 	// Per CPU maps return values larger than the size in the spec
 	fullValueSize int
+	// randKey used for kernel < 4.4.132 that didn't support nil key
+	randKey []byte
```

Comment:
Why are these part of `Map`?

Reply:
It was to keep the random context per map, but it's optional